src/formatters.ts

```diff
@@ -8,6 +8,8 @@ export interface Formatter {
 	};
 }
 
+export type FormatterName = "Biome" | "deno fmt" | "dprint" | "Prettier";
```

Comment:
> rename the accepted names to `"biome"`, `"deno"`, `"dprint"`, and `"prettier"` 

Agreed, that would be better. Especially once they're used as a top-level input.

I think it's fine to make this change now. The package is still 0.x and _very_ early stage.

Reply:
I've renamed them at https://github.com/JoshuaKGoldberg/formatly/pull/71/commits/d93176113bb88a0b6fa4c170ad838c7f83eec598